### PR TITLE
Update SLUS-20591_2FF8E31F.pnach

### PR DESCRIPTION
**Pull request by Joshua-1248**
Adds deinterlacing option to DragonBall Z - Budokai 1

### CODE REVIEW

File: patches/SLUS-20591_2FF8E31F.pnach [hunk 1]
```diff
@@ -14,4 +14,7 @@ patch=1,EE,0034de50,word,3f400000 //3f800000
 patch=1,EE,0034deb0,word,3f400000 //3f800000
 patch=1,EE,00355640,word,3f400000 //3f800000
 
-
+[Deinterlacing Patch]
```
Comment (Mrlinkwii) [inline patches/SLUS-20591_2FF8E31F.pnach:17]: needs the special notifier for pcsx2 to change the settings , 

 gsinterlacemode=1
 
 see https://forums.pcsx2.net/Thread-Sticky-Important-Patching-Notes-1-7-4546-Pnach-2-0

**Review by Mrlinkwii — APPROVED** [review]
assumed tested

File: patches/SLUS-20591_2FF8E31F.pnach [hunk 2]
```diff
@@ -14,4 +14,9 @@ patch=1,EE,0034de50,word,3f400000 //3f800000
 patch=1,EE,0034deb0,word,3f400000 //3f800000
 patch=1,EE,00355640,word,3f400000 //3f800000
 
-
+[No-Interlacing]
+author=Josh_7774
+description=Attempts to disable interlaced offset rendering.
+gsinterlacemode=1
+////Disable Interlacing Instruction
```
Comment (refractionpcsx2) [inline patches/SLUS-20591_2FF8E31F.pnach:21]: any reason for 4 / ? 2 is enough :)

Comment (Joshua-1248) [reply patches/SLUS-20591_2FF8E31F.pnach]: Fixed.

**Pull request merged**